smoketest/scripts/cli/test_load-balancing_haproxy.py

```diff
@@ -521,5 +521,53 @@ def test_10_lb_reverse_proxy_http_compression(self):
         with self.assertRaises(ConfigSessionError) as e:
             self.cli_commit()
 
+    def test_11_lb_haproxy_timeout(self):
+        t_default_check = '5'
+        t_default_client = '50'
+        t_default_connect = '10'
```

Comment:
CLI default values can be queried by xml ref tree - see SSH testcases as example. Just a hint for future implementations.